db/schema.rb

```diff
@@ -1,100 +1,99 @@
-# encoding: UTF-8
 # This file is auto-generated from the current state of the database. Instead
 # of editing this file, please use the migrations feature of Active Record to
 # incrementally modify your database, and then regenerate this schema definition.
 #
-# Note that this schema.rb definition is the authoritative source for your
-# database schema. If you need to create the application database on another
-# system, you should be using db:schema:load, not running all the migrations
-# from scratch. The latter is a flawed and unsustainable approach (the more migrations
-# you'll amass, the slower it'll run and the greater likelihood for issues).
+# This file is the source Rails uses to define your schema when running `rails
+# db:schema:load`. When creating a new database, `rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160316132743) do
+ActiveRecord::Schema.define(version: 2016_03_16_132743) do
 
-  create_table "entities", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 ROW_FORMAT=COMPACT" do |t|
-    t.string  "name",                      default: "",    null: false
-    t.string  "legal_name",                default: "",    null: false
-    t.text    "legal_other", limit: 65535
-    t.text    "address",     limit: 65535
-    t.text    "telephone",   limit: 65535
-    t.text    "fax",         limit: 65535
-    t.text    "email",       limit: 65535
-    t.text    "dp_reg",      limit: 65535
-    t.text    "dp_contact",  limit: 65535
-    t.text    "signatory",   limit: 65535
-    t.boolean "is_eea",                    default: false, null: false
-    t.text    "country",     limit: 65535,                 null: false
-  end
-
-  add_index "entities", ["name"], name: "entityname", unique: true, using: :btree
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "pg_stat_statements"
+  enable_extension "plpgsql"
+  enable_extension "uuid-ossp"
 
-  create_table "entity_transfers", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 ROW_FORMAT=COMPACT" do |t|
-    t.integer "entity_id",    limit: 5,       null: false
-    t.integer "transfer_id",  limit: 5,       null: false
-    t.boolean "is_recipient", default: false, null: false
+  create_table "countries", id: false, force: :cascade do |t|
+    t.text "country"
+    t.text "code"
+    t.text "lat"
+    t.text "lon"
```

Comment:
Where did that come from? I don't see it in the source.

Reply:
I generated the schema.rb file based on the migration of data over from MySQL to Postgresql via AWS tools. The countries table did exist in the MySQL database, even if there's not reference to it in the previous schema.rb file.